### PR TITLE
feat: add navigation for action button

### PR DESCRIPTION
**Pull request by niekvanstaveren**

### CODE REVIEW

**Comment by niekvanstaveren** [comment]
@eliseeborn and @haxxmaxx Please start having a look. Addition of navigation and I moved some code around as well. For now still a draft since I want to add some unit test still.

**Comment by niekvanstaveren** [comment]
@Caele Basic navigation functionality. However it assumes that firstsheet and lastsheet will be present. I think we now said that they won't right?

File: src/utils/navigation-actions.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    requiredInput: ['story'],
+  },
+  {
+    label: 'Open a website / eMail',
+    value: 'openWebsite',
+    navigationCall: async ({ websiteUrl, sameWindow }) => {
+      try {
+        if (websiteUrl) {
+          const url =
+            websiteUrl.startsWith('https://') || websiteUrl.startsWith('http://') || websiteUrl.startsWith('mailto://')
```
Comment (miralemd) [inline src/utils/navigation-actions.js]: `startsWith` is a fairly new method on `String` - will not work in IE, might wanna polyfill this

Comment (miralemd) [reply src/utils/navigation-actions.js]: or use a regex: `websiteUrl.match(/^(https?|mailto):\/\//)`

**Comment by Caele** [comment]
> @Caele Basic navigation functionality. However it assumes that firstsheet and lastsheet will be present. I think we now said that they won't right?

no, assume that those will not be part of the API

File: src/utils/navigation-actions.js (hunk 1 above)
Comment (niekvanstaveren) [reply src/utils/navigation-actions.js]: Ah thanks! Will change to regex

File: src/utils/navigation-actions.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  },
+  {
+    label: 'Go to first sheet',
+    value: 'firstSheet',
+    navigationCall: async ({ senseNavigation }) => {
+      try {
+        await senseNavigation.firstSheet();
+      } catch (error) {
+        // no-op
+      }
```
Comment (Caele) [inline src/utils/navigation-actions.js]: why use a try/catch?

**Pull request merged**